### PR TITLE
SOC-1764 | Fix title when hostname is present

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-1764

### CODE REVIEW

**Comment by gabrys** [comment]
LGTM. Better than before, but still not perfect.

**Comment by rogatty** [comment]
How about a unit test?

**Comment by kvas-damian** [comment]
@rogatty they're still passing ;)

**Comment by rogatty** [comment]
Shouldn't be too hard to add one more test to /test/specs/server/lib/Utils.js

**Comment by kvas-damian** [comment]
Already working on that ;)

**Comment by kvas-damian** [comment]
@rogatty see https://github.com/Wikia/mercury/commit/7f34c8a32a98d1f0e2e4bfcd23883b4a8dc03828

File: test/specs/server/lib/Utils.js [hunk 1]
```diff
@@ -371,3 +371,37 @@ test('redirectToCanonicalHostIfNeeded', function () {
...
+			htmlTitleTemplate: 'sandbox - $1 - Muppet Wiki - Wikia',
+			displayTitle: '',
+			expected: 'sandbox - Muppet Wiki - Wikia'
+		},
+		{
+			htmlTitleTemplate: '$1 - Wikia',
+			displayTitle: '',
+			expected: 'Wikia'
+		},
+	];
```
Comment (rafalkalinski) [inline test/specs/server/lib/Utils.js]: How about:
```{
			htmlTitleTemplate: '$1 - Wikia',
			displayTitle: '$1',
			expected: 'Wikia'
		},```
???

**Comment by rafalkalinski** [comment]
LGTM

**Comment by rogatty** [comment]
Thanks! LGTM

**Pull request merged**